components/_patterns/02-molecules/card/01-card.twig

```diff
@@ -1,51 +1,126 @@
-<div class="card{% for modifier in card_modifiers %} card--{{ modifier }}{% endfor %}{% if card_blockname %} {{ card_blockname }}__card{% endif %}">
-  {% if card_img %}
-    <div class="card__img">
-      <img src="{{ card_img }}" />
-    </div>
-  {% elseif image %}
-    <div class="card__img">
-      {{ image }}
-    </div>
+{#
+/**
+ * Available variables:
+ * - card_base_class - base classname of the wrapper
+ * - card_modifiers - array of modifiers to add to the base classname of the wrapper
+ * - card_blockname - blockname prepended to the base classname of the wrapper(s) and each component
+ *
+ * - card_label_base_class - base classname of the card label
+ * - card_label - the card label
```

Comment:
Seems this should be listed first as the *_base_class normally comes after the source data.